### PR TITLE
Merge the TpcRS parameters for Run18 FXT26p5 to StarDb/Calibrations/tpc/

### PR DESCRIPTION
**Pull request by zhux97**
The new T0 parameters are tuned by Yue-Hang Leung and Yuri Fisyak.

### CODE REVIEW

**Comment by iraklic** [comment]
Should we somehow keep record of based on what results have the parameters been produced? Like a quick snapshot of final plots which produced these params. Otherwise the files itself looks legit and it Yuri glanced over this then I'll just click approve once he does. Is there any practice of keeping track of such results on github? Perhaps a link to a permanent drupal page?

**Comment by zhux97** [comment]
> Should we somehow keep record of based on what results have the parameters been produced? Like a quick snapshot of final plots which produced these params. Otherwise the files itself looks legit and it Yuri glanced over this then I'll just click approve once he does. Is there any practice of keeping track of such results on github? Perhaps a link to a permanent drupal page?

Hi Irakli, please find the plots in Yue-Hang's slides regarding this file, 
https://drupal.star.bnl.gov/STAR/system/files/7p2embqa.pdf

**Comment by iraklic** [comment]
Just to reiterate what I also wrote in the email. The T0 correction event-by-event is for the data, that should have nothing to do with the simulation. The TpcRS parameters need to be tuned time to time, and yes, they need to match properly calibration data, which on its side needs the vertex correction. It looks like the TpcRS has been recalibration for the data and you seem to be satisfied with the result. I am sure Yuri is aware of this and will also look at the results. With this said I will approve the new and improved parameters to be put in the repo.

**Review by iraklic — APPROVED** [review]
See my comments in the comment section.

File: StarDb/Calibrations/tpc/TpcResponseSimulator.20180605.071409.C [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+//
+// Revision 1.3  2010/04/16 19:31:19  fisyak
+// Intermidiate version
+//
+// Revision 1.3  2010/04/04 23:14:33  fisyak
+// Add Row Correction
+//
+// Revision 1.2  2010/04/01 22:17:57  fisyak
+// Freeze version W
+//
```
Comment (plexoos) [inline StarDb/Calibrations/tpc/TpcResponseSimulator.20180605.071409.C:68]: This is a new file. Is this embedded history really relevant?

```suggestion
```

**Comment by zhux97** [comment]
Hi all, Yuri will send in a new version of this file to git, I will close this pull request. Thanks to everyone for comments.

**Pull request closed without merging**